package.json

```diff
@@ -174,19 +174,23 @@
     "eslint": "^4.8.0"
   },
   "dependencies": {
-    "@folio/stripes-components": "^2.0.20",
     "@folio/stripes-form": "^0.8.2",
     "@folio/stripes-smart-components": "^1.4.0",
     "eslint": "^4.8.0",
     "lodash": "^4.17.5",
+    "moment": "^2.22.2",
+    "moment-timezone": "^0.5.20",
     "prop-types": "^15.5.10",
     "query-string": "^5.0.0",
+    "react": "^16.4.1",
+    "react-dom": "^16.4.1",
     "react-router-dom": "^4.1.1",
     "redux-form": "^7.3.0",
     "style-loader": "^0.20.1",
     "uuid": "^3.1.0"
   },
   "peerDependencies": {
+    "@folio/stripes-components": "^2.0.0",
```

Comment:
Pretty sure you want to keep stripes-components as a regular ol' dependency and move react and react-dom under peer-dependencies. Peer-deps look outward, at how your module interacts with the surrounding ecosystem. Regular deps look inward, at the requirements for how your app functions internally.

Reply:
Thanks zak, i'll do it in the next commit today. thank you.